projects/js-packages/publicize-components/src/hooks/use-media-restrictions/index.js

```diff
@@ -1,11 +1,13 @@
 import { useRef, useMemo } from '@wordpress/element';
-import { DEFAULT_RESTRICTIONS, RESTRICTIONS } from './restrictions';
+import { DEFAULT_RESTRICTIONS, GLOBAL_MAX_SIZE, RESTRICTIONS } from './restrictions';
```

Comment:
I can't remember why but I think this was previously removed as it wasn't needed. It might have been that we set it on all the restrictions, in which case it probably does make sense to double check here.

Reply:
Yeah, I put it back, because the with the inner objects in restrictions, the default restrictions does not always do the job. Using default values when extracting from `videoLimits` makes sure it's enough

Reply:
Fair enough. Why is that? Is it because we haven't added the defaults to each of the services?

Reply:
It's because we use the Defaults, when there is nothing for a service, but if we have restrictions for let's say Facebook, which have no aspect ratio limitations, that won't get in from the Defaults. We could write a function that deep merges by keys too, but I think this should do it for now.